### DISCUSSION TITLE
Updating stats-overview-widget values when Table updated

### DISCUSSION THREAD

**Discussion by kuchienkz** [filamentphp/filament#8278] [Help] [answered]
Is there a way to update stats-overview-widget values when Table updated by Filters ? I'm trying to show some summary values using stats-overview-widget, based on current Table records.

**Comment by leandrocfe** [comment, marked as answer]
> Is there a way to update stats-overview-widget values when Table updated by Filters ? I'm trying to show some summary values using stats-overview-widget, based on current Table records.

V2 - https://filamentphp.com/community/how-to-refresh-widgets-when-table-actions-are-fired
